### DISCUSSION TITLE
Code for "A Simple Approach to Differentiable Rendering of SDFs"

### DISCUSSION THREAD

**Discussion by AntonioDOrazio** [mitsuba-renderer/mitsuba3#1188] [Q&A]
Good morning, I was reading this new paper called [A Simple Approach to Differentiable Rendering of SDFs](https://arxiv.org/abs/2405.08733v1). I would like to ask if you plan to release the code/update Mitsuba to support this new approach to differentiable rendering on SDFs, and if there is a tentative schedule for it, as I would be very interested to use your new findings for a project I'm currently working on.

Thank you!

**Comment by merlinND** [comment]
Hello @AntonioDOrazio,

I think that your best bet would be to email the authors of the paper, as it's not likely that they are all monitoring this discussion space.
